identity/handler.go

```diff
@@ -30,6 +30,12 @@ func SelectIdentity(dir string, nodeKey string) (id *dto.Identity, err error) {
 		return
 	}
 
+	return
+}
+
+func CreateIdentity(dir string) (id *dto.Identity, err error) {
+	handler := NewIdentityHandler(dir)
```

Comment:
Use DI pattern. All dependencies should be injected for top layer, avoid constructing them in the middle of functions

Reply:
Same in `SelectIdentity()`, and we have situation then handler is constructed 2 times